### DISCUSSION TITLE
Doc clarification: createWeb3Modal and ERC-1193 provider

### DISCUSSION THREAD

**Discussion by wighawag** [WalletConnect/walletconnect-monorepo#4359] [Developer Support]
HI,
I am reading the doc for v2 and I am a bit confused.

I am not using ethers nor I am using wagmi and neither react or vue, or any lib mentioned in your doc

I see that we can use the ethereum provider option (ERC-1193) but it does not seems to have all the option of `createWeb3Modal` 

In particular the doc do not mention email wallet or onramp in that section

Also, wondering why there is ethers version of createWeb3Modal but not viem (even though you have wagmi)

**Comment by overplex-eth** [comment]
The existence of an ethers version of createWeb3Modal but not for Vue (assuming you meant Vue instead of "viem"), despite having wagmi, likely reflects the broader adoption or demand within the developer community for certain libraries. ethers.js is a widely used library for interacting with Ethereum, which might explain its direct support. In contrast, Vue-specific integrations might rely on more general-purpose libraries or require a community-driven effort to provide equivalent functionality.

**Comment by wighawag** [comment]
No I meant viem, what wagmi uses under the hood.
